src/webview/js/js.js

```diff
@@ -740,9 +740,22 @@ const handleLongPress = (target: Element) => {
 
   hasLongPressed = true;
 
+  if (target.closest('.reaction-list')) {
+    const reactionNode = target.closest('.reaction');
+    if (!reactionNode) {
+      return;
+    }
```

Comment:
Also from our voice chat today: why have this `return` case at all? How about just starting with `target.closest('.reaction')` and not looking for the `.reaction-list`? We don't seem to actually use the latter once we find it.